<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgFileDlg</class>
 <widget class="QDialog" name="ImgFileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Specify image file format</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okBtn">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>number of row intersections</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="typeBox"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Base name for left image files</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lBaseEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Base name for right image files</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Image type</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Calibration name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="rBaseEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Image file directory</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="pathEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseBtn">
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>number of column intersections</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="rowNumEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="colNumEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>size of the squares (mm)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="scaleEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
